tools/osv-scanner/cmd/osv-scanner/main.go

```diff
@@ -10,56 +10,67 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/BurntSushi/toml"
+	"github.com/g-rath/osv-detector/pkg/lockfile"
 	"github.com/google/osv.dev/tools/osv-scanner/internal/osv"
 	"github.com/google/osv.dev/tools/osv-scanner/internal/output"
 	"github.com/google/osv.dev/tools/osv-scanner/internal/sbom"
-
-	"github.com/g-rath/osv-detector/pkg/lockfile"
 	"github.com/urfave/cli/v2"
+	"golang.org/x/exp/slices"
 )
 
+const osvScannerConfigName = "osv-scanner.toml"
+
+var globalConfig *Config
+
 // scanDir walks through the given directory to try to find any relevant files
```

Comment:
Please elaborate a little more in the docstring as to what's considered relevant.